src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -54,24 +54,32 @@ public async Task OnGetAsync(string id)
             var assemblyModel = await assemblyRepository.ReadAssemblyContentAsync(id);
             var renderer = new HTMLRendererAPIV();
             AssemblyModel = renderer.Render(assemblyModel.Assembly).ToArray();
-            var comments = await commentRepository.FetchCommentsAsync(id);
-
             Comments = new Dictionary<string, List<CommentModel>>();
-            foreach (var comment in comments)
+
+            var assemblyComments = await commentRepository.FetchCommentsAsync(id);
+            if (assemblyComments != null)
```

Comment:
Why would it be null?

Reply:
Currently, a blob isn't created to store review comments until a comment is uploaded, so `FetchCommentsAsync()` will return null the first time a review is opened.

Reply:
Return an empty object then.